File: code/be/src/main/java/com/music/application/be/modules/user/dto/ProfileDTO.java
package com.music.application.be.modules.user.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.RequiredArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProfileDTO {
    private String username;
    private String email;
    private String phone;
    private String avatar;
    private int favoriteSongCount;
    private int followedArtistCount;
    private int playlistCount;
}

File: app/src/main/java/com/example/musicapplicationse114/model/ProfileDto.kt
package com.example.musicapplicationse114.model

data class ProfileDto(
    val username: String,
    val email: String,
    val phone: String,
    val avatar: String,
    val favoriteSongCount: Int,
    val followedArtistCount: Int,
    val playlistCount: Int
)

